Tests/scripts/collect_tests_and_content_packs.py

```diff
@@ -793,6 +794,26 @@ def extract_matching_object_from_id_set(obj_id, obj_set, server_version='0'):
     return None
 
 
+def get_packs_from_landing_page(branch_name: str) -> set:
+    """
+    Parses the changes made by the current branch to the landing page sections file into modified packs.
+    These modified packs will have the metadata files changed.
+    Args:
+        branch_name: The name of the branch to compare changes from origin/master
+
+    Returns:
+        A set of packs that needs to be updated.
+    """
+    with open(LANDING_PAGE_SECTIONS_JSON_PATH, 'r') as file:
+        landing_page_sections_content = json.load(file)
```

Comment:
lets change this to something like `landing_page_sections_content = json.load(file).keys()` so we wont have to call `.keys()` in every loop iteration